examples/booking_app/test/feature_hotel_favorites/view/hotel_favorites_test.dart

```diff
@@ -0,0 +1,32 @@
+import 'package:booking_app/feature_hotel_favorites/blocs/hotel_favorites_bloc.dart';
+import 'package:booking_app/feature_hotel_favorites/views/hotel_favorites_page.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:provider/provider.dart';
+import 'package:rx_bloc/rx_bloc.dart';
+
+import '../mock/hotel_favorites_mock.dart';
+
+void main() {
+  group('HotelFavoritesPage', () {
+    testWidgets('Error retry widget', (tester) async {
+      final mockHotelFavBloc = hotelFavoritesMockFactory(
+        favoriteHotels: Result.error(Exception('Error')),
+      );
+      await tester.pumpWidget(MultiProvider(
+        providers: [
+          Provider<HotelFavoritesBlocType>.value(value: mockHotelFavBloc),
+        ],
+        child: const MaterialApp(
+          home: Scaffold(body: HotelFavoritesPage()),
+        ),
+      ));
+      await tester.pump(const Duration(seconds: 1));
```

Comment:
Can't we use this method instead ?

https://api.flutter.dev/flutter/flutter_test/WidgetTester/pumpAndSettle.html

Reply:
@StanevPrime  I've tried this approach, however since there aren't any animations to settle this fails.

The implementation of `HotelAnimatedListView` should have animations and it does when the app is running.  However, they aren't rendered in the tests even after waiting for them for a long time. I suspect something related to the underlying package used (`animated_stream_list`) is making those kind of issues. When the widget from that package is replaced by a `ListView`, everything works as intended.